### PR TITLE
Update d1.py to make it clearest possible instance

### PR DESCRIPTION
**Pull request by joesho112358**

### CODE REVIEW

File: d1.py [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+# clearly, black magic
+with open("input.txt", "r+") as file1262:
+    for l in file1262:
+        all_values_which_are_determined_to_be_digits = is_digit_in_string(l)
+        if len(all_values_which_are_determined_to_be_digits) == 1 :
+            all_the_things.append(all_values_which_are_determined_to_be_digits[0] * 11)
         else:
-            x = int(str(values[0]) +  str(values[-1]))
-            all.append(x)
+            temporaryThingToStoreIntoTheVariableAll = int(str(all_values_which_are_determined_to_be_digits[0]) +  str(all_values_which_are_determined_to_be_digits[-1]))
```
Comment (newtewt) [inline d1.py]: line is too long

Comment (joesho112358) [reply d1.py]: fixed

**Comment by newtewt** [comment]
Since you have interest in this project and I'm to busy. Do you want to take over ownership @joesho112358

**Comment by joesho112358** [comment]
I only know FORTRAN

**Pull request closed without merging**